src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should avoid using units like `vmin` for padding, margin, border, etc. It's better to use `px`, `em`, or `rem` for these properties to ensure consistency across different screen sizes and resolutions.